.github/readmes/_using-the-graphql-api-typegraphql-crud.md

```diff
@@ -0,0 +1,139 @@
+## Using the GraphQL API
+
+`typegraphql-prisma` emits the generated TypeGraphql classes to `@generated/typegraphql-prisma` that is located in the `node_modules` whenever `npx prisma generate` is invoked. It also generates a number of model classes, enums and CRUD resolvers and relations resolver based on your `schema.prisma` file. CRUD resolvers support the following operations that are 1:1 matching with the `PrismaClient` API:
+
+- create
+- update
+- delete
+- findUnique
+- findFirst
+- findMany
+- updateMany
+- deleteMany
+- upsert
+- aggregate
+- groupBy
+
+Below are example operations that you can send to the API using the GraphQL Playground. You can explore other operations in the Docs section of the GraphQL Playground.
```

Comment:
When referring to button names of UI elements, I think it's nice to highlight these in bold. (This creates a visual "anchor" for people to quickly recognize it on a UI as well). For reference: I've used this approach quite a bit in this article where I described a lot of workflows in a browser UI: https://dev.to/prisma/how-to-setup-a-free-postgresql-database-on-heroku-1dc1

```suggestion
Below are example operations that you can send to the API using the GraphQL Playground. You can explore other operations in the **Docs** section of the GraphQL Playground.
```